packages/block-library/src/text-columns/block.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "core/text-columns",
+	"icon": "columns",
+	"category": "layout",
+	"attributes": {
+		"content": {
+			"type": "array",
+			"source": "query",
+			"selector": "p",
+			"query": {
+				"children": {
+					"type": "string",
+					"source": "html"
+				}
+			},
+			"default": [ { }, { } ]
```

Comment:
Nit: We should consider if/what JSON style standards we want. If it follows JavaScript objects, these padded empty objects would be invalid:

>No filler spaces in empty constructs (e.g., {}, [], fn()).

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing

Reply:
It's probably code reformat I used in my IDE :)